index.html

```diff
@@ -0,0 +1,267 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
```

Comment:
- [OPTIONAL] I think it would be better to use more descriptive title, for example "your name + 's + Portfolio" 💯

Reply:
Thank you very much